templates/terraform/constants/region_backend_service.go.erb

```diff
@@ -49,28 +49,28 @@ func validateNonInternalBackendServiceBackends(backends []interface{}, d *schema
 		if v, ok := backend["capacity_scaler"]; ok && v != nil {
 			sum += v.(float64)
 		} else {
-			return fmt.Errorf("capacity_scaler is required for each backend in non-INTERNAL backend service")
+			return fmt.Errorf("capacity_scaler is required for each backend in managed backend service")
 		}
 	}
 	if sum == 0.0 {
-		return fmt.Errorf("non-INTERNAL backend service must have at least one non-zero capacity_scaler for backends")
+		return fmt.Errorf("managed backend service must have at least one non-zero capacity_scaler for backends")
 	}
 	return nil
 }
 
-// If INTERNAL, make sure the user did not provide values for any of the fields that cannot be sent.
+// If INTERNAL or EXTERNAL, make sure the user did not provide values for any of the fields that cannot be sent.
 // We ignore these values regardless when sent to the API, but this adds plan-time validation if a
 // user sets the value to non-zero. We can't validate for empty but set because
 // of how the SDK handles set objects (on any read, nil fields will get set to empty values)
-func validateInternalBackendServiceBackends(backends []interface{}, d *schema.ResourceDiff) error {
+func validateNonManagedBackendServiceBackends(backends []interface{}, d *schema.ResourceDiff) error {
 	for _, b := range backends {
 		if b == nil {
 			continue
 		}
 		backend := b.(map[string]interface{})
-		for _, fn := range backendServiceOnlyNonInternalFieldNames {
+		for _, fn := range backendServiceOnlyManagedFieldNames {
 			if v, ok := backend[fn]; ok && !isEmptyValue(reflect.ValueOf(v)) {
-				return fmt.Errorf("%q cannot be set for INTERNAL backend service, found value %v", fn, v)
```

Comment:
I'm seeing a failure in `TestAccComputeRegionBackendService_withBackendInternal` where we don't see this error anymore. I think we just need to edit the test to expect the new message.

Reply:
Sorry, should have run all the region backend services acceptance tests since I changed files that impact those resources too. Fixed and confirmed by running all those acceptance tests now.